### DISCUSSION TITLE
Third party - styled-system support

### DISCUSSION THREAD

**Discussion by rmcarias** [chakra-ui/chakra-ui#5414] [General]
I read the docs that mention Chakra is based on styled-system under the hood. That said, wondering if you all have had a chance to review this thread on their Github https://github.com/styled-system/styled-system/issues/1547 - and how it will impact Chakra itself (if any).

**Comment by primos63** [comment]
This has no impact to the project as Chakra is not based on the `styled-system` repo. Chakra uses Emotion, @emotion/react and @emotion/styled. There is a Chakra `styled-system` [package](https://github.com/chakra-ui/chakra-ui/tree/main/packages/styled-system) in our repo which we maintain. There's a difference between being patterned after something and using it. Perhaps the docs should be clearer on that.

**Reply by rmcarias** [reply]
Got it! Great to know!
